### ISSUE TITLE
Getting "application/mbox charset=gzip" for .mbox.gz files

### ISSUE THREAD

**Issue by acidus99** [michael-lazar/jetforce#75]
I have a GZIP compressed mbox file named `entire-archive.mbox.gz` on my capsule. I'm using a stock version of Jetforce 0.9.0.

When someone accessing this file, Jetforce returns a META value of `application/mbox charset=gzip` for the response, as shown in the screen shot of Lagrange below:

<img width="241" alt="image" src="https://github.com/michael-lazar/jetforce/assets/4118560/9e21285e-6375-4ce2-982b-13be9b3fd241">

 `charset=gzip` is not an IANA defined charset 😂.  My expectation is for Lagrange to serve the response with a a mimetype of `application/gzip`.

I don't think this is bad issue necessarily. Most gemini clients will prompt the user to download the response if the mimetype is unrecognized or not text. So the net result to the user is the same: they will download the file. However I just thought it was odd to get such an odd, incorrect Mimetype out of a stock install of Jetforce.

Here is an mbox.gz URL you can test with:
```
gemini://gemi.dev/gemini-mailing-list/entire-archive.mbox.gz
```

**Comment by michael-lazar** [comment]
Yes, you're totally right.

Python will decipher the mimetype for a file based on the extension, and return a tuple of `(mimetype, encoding)` [0]. I must have just confused encoding with charset at the time, or assumed that they meant the same thing for non-text responses.

The code in jetforce is pretty straightforward [1]. I'll play around with it and see if I can find a better solution.

[0] https://docs.python.org/3/library/mimetypes.html#mimetypes.guess_type
[1] https://github.com/michael-lazar/jetforce/blob/e08aa4bb0d300a503ae721cedc443e78f94ba08e/jetforce/app/static.py#L265C12-L272C54

**Comment by michael-lazar** [comment]
Fixed in [v0.10.0](https://github.com/michael-lazar/jetforce/releases/tag/v0.10.0)!
